projects/vdk-core/src/vdk/internal/builtin_plugins/run/cli_run.py

```diff
@@ -143,14 +144,41 @@ def __log_exec_result(self, execution_result: ExecutionResult) -> None:
 
     @staticmethod
     def __log_short_exec_result(execution_result):
+        def extract_relevant_lines(step):
+            out = []
+            if step.exception:
+                call_list = traceback.format_tb(step.exception.__traceback__)
```

Comment:
Is it possibel something to fail? is `__traceback__` always available. 

I am asking because If something fails, it's better to still log something (with less detail) than to crash teh program

Reply:
Calling `traceback.format_tb(None)` just returns an empty list. `__traceback__` should always be available, but I think we're safe if it's not.